index.html

```diff
@@ -18,7 +18,10 @@ <h2 id="talk-name"></h2>
     <h4 class="fixed-text">The format of the talk is: </h4>
     <h3 id="talk-type"></h3>
 
-    <div id="button">Click here to generate a new topic!</div>
+    <div id="button" class="button">Click here to generate a new topic!</div>
+    <br/>
+    <div id="floating-point-button" class="button">Click here to adjust the floating point value for the output</div>
+    <input type="text" id="floating-point-input"></input>
```

Comment:
is the input value consumed in JS already?  or is that a subsequent PR?

Reply:
It is consumed or should be. The JS requirements are not blown away by Sean so this fixes the front end issue.

Reply:
hahaha.  👍

Reply:
<img width="446" alt="this_page_says" src="https://user-images.githubusercontent.com/1323521/39667745-0c893abc-507a-11e8-8d0f-ac9a476d2e47.png">

validation is a good idea!!  can you adjust the maximum to be higher?  ...maybe like 2.0?

Reply:
oh!  and the minimum to be 0.01?

exactly 0 seems to break it.

Reply:
Updated

Reply:
I'm PR'ing into this to make this "prettier"

Reply:
:+1: